libcaliptra/inc/caliptra_api.h

```diff
@@ -22,5 +22,8 @@ int caliptra_upload_fw(struct caliptra_buffer *fw_buffer);
 // Read Caliptra FIPS Version
 int caliptra_get_fips_version(struct caliptra_fips_version *version);
 
+// Stash Measurement command
+int caliptra_stash_measurement(struct caliptra_stash_measurement_req *req, struct caliptra_stash_measurement_cpl *cpl);
```

Comment:
(maybe for a different PR)

The error codes returned by this library are a bit confusing. It seems like they might be a Caliptra error code (from https://github.com/chipsalliance/caliptra-sw/blob/main/error/src/lib.rs) or a linux errno. Maybe we should align on just using caliptra error codes and then define a range of error codes that are client-side failures (e.g. an invalid checksum or invalid input arguements)

Reply:
At the top level they should (or have been) only unix-style error codes. caliptra_mbox_read_status returns Caliptra error codes, but does not return them to the caller.

Reply:
Details about the return codes used and what they indicate should be added to caliptra_api.h. Right now a caller has no way to know how to interpret the return value.

Reply:
https://github.com/chipsalliance/caliptra-sw/issues/678

Reply:
Sounds good, thanks